python/mlc_chat/compiler/compile.py

```diff
@@ -83,15 +83,17 @@ def _emit_metadata(metadata):
 
 
 def _attach_variable_bounds(mod, model_config):
+    tir_bound_map = {}
+    tir_bound_map["seq_len"] = model_config.prefill_chunk_size
+    if model_config.context_window_size != -1:
```

Comment:
I wonder if this bound is also applied to mistral; should we do an else branch after `sliding_window` to set bound on `total_seq_len`? As of now, I think we do generate `context_window_size` in `model_config`, but it is not bounded since we do not perform shift and re-encode. Let me know if I'm missing something.

Reply:
It makes sense, fixed now, thanks @CharlieFRuan !